### PR TITLE
make number of images over 8 unselectable

### PR DESCRIPTION
**Pull request by zoebarth**
image

After the user selects 8 images, make it impossible to select any more, and alert the user.

### CODE REVIEW

File: frontend/src/ExploreView.module.css [hunk 1]
```diff
@@ -77,6 +77,17 @@
...
+
+.unselectable {
+  opacity: 0.3;
+  pointer-events: none;
+}
+
+.p {
+  height: 25px;
+  margin: -25px 0 0;
+  text-align: center;
```
Comment (Cindyzhang977) [inline frontend/src/ExploreView.module.css:92]: nit: can you make the margin a little more between the text and the pictures? Also could you make the text color `var(--gray)` (the gray set in `App.css`) so the contrast with the white background isn't too much?

File: frontend/src/ExploreView.js [hunk 2]
```diff
@@ -94,7 +97,11 @@ function Explore() {
               initialAttractions.map((attraction, index) => (
                 <Card
                   className={`${styles.attractionContainer} ${
-                    attraction.selected ? styles.selectedImage : ''
+                    attraction.selected
+                      ? styles.selectedImage
+                      : selectedAttractions.length < 8
+                      ? ''
+                      : styles.unselectable
```
Comment (mecolmg) [inline frontend/src/ExploreView.js:104]: nit: this is hard to read and understand. Also, you're possibly adding an extra space unnecessarily. Maybe something like the following, or consider using something like the [`classnames package`](https://www.npmjs.com/package/classnames#alternate-bind-version-for-css-modules-)

```js
className={`${styles.attractionContainer}${
    attraction.selected ? ` ${styles.selectedImage}` : ''
  }${
    selectedAttractions.length >= 8 ? ` ${styles.unselectable}` : ''
  }`}
```

File: frontend/src/ExploreView.module.css (hunk 1 above)
Comment (zoebarth) [reply frontend/src/ExploreView.module.css:92]: Done

File: frontend/src/ExploreView.js (hunk 2 above)
Comment (zoebarth) [reply frontend/src/ExploreView.js:104]: The problem with what you wrote is that if the attraction is selected, I don't want it to be unselectable

Comment (zoebarth) [reply frontend/src/ExploreView.js:104]: The structure i wrote should mimic:
if (selected) {
} else {
  if(length < 8) {
  } else {
  }
}

**Pull request merged**